src/css/profile/mobile/common/calendar.less

```diff
@@ -0,0 +1,116 @@
+body {
+	margin: 0;
+}
```

Comment:
Why do you set the margin of the body here?

Reply:
There was margin in the sample we were testing. I'll remove it.